src/System.Private.Xml/tests/XmlResolver/System.Xml.XmlResolver.Tests/XmlPreloadedResolverGetEntity.cs

```diff
@@ -159,11 +159,11 @@ private static IEnumerable<(string PublicId, string SystemId, string SourcePath)
                 ("-//Netscape Communications//DTD RSS 0.91//EN", "rss-0.91.dtd"),
             };
 
-            string dtdFolderRoot = @"Utils\DTDs\";
+            string dtdFolderRoot = "Utils/DTDs/";
```

Comment:
those can stay as they were

Reply:
In the end I felt it was inconsistent to be using `Path.Combine` to form the `XHTML10/no_comments` etc on the lines immediately after this but to be using slashes in the strings here, so I've now modified this to use `Path.Combine` just like everywhere else that wrangles file paths in this test. (Otherwise you ended up with weirdly inconsistent looking paths such as `Utils\DTDs/XHTML10/no_comments`. Using `Path.Combine` ubiquitously means they all look right for whatever platform you're running on.)